client/i18n/locales/english/intro.json

```
@@ -3515,12 +3508,12 @@
       "review-typescript": {
         "title": "Typescript Review",
         "intro": [
-          "Review the Typescript concepts to prepare for the upcoming quiz."
+          "Review Typescript concepts to prepare for the upcoming quiz."
         ]
       },
       "quiz-typescript": {
         "title": "TypeScript Quiz",
-        "intro": ["Test what you've learned in this quiz on TypeScript."]
+        "intro": ["Test what you've learned on Typescript with this quiz."]
```

```suggestion
        "intro": ["Test what you've learned on TypeScript with this quiz."]
```

It *is* capitalised like this.